extensions/wikia/ThemeDesigner/UploadBackgroundFromFile.class.php

```diff
@@ -8,24 +9,24 @@ class UploadBackgroundFromFile extends UploadFromFile {
 	const FILESIZE_ERROR = 20;
 	const FILETYPE_ERROR = 21;
 
-	public function verifyUpload(){
+	public function verifyUpload() {
 		$this->getTitle(); //will fill in final destination and extension
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)